blenc.c

```diff
@@ -552,15 +553,16 @@ b_byte *php_blenc_decode(void *input, unsigned char *key, int in_len, int *out_l
 static unsigned char *php_blenc_gen_key()
 {
 
-	int sec = 0 , usec = 0;
+	time_t sec = 0;
+    long int usec = 0;
 	struct timeval tv;
 	char *retval = NULL, *tmp = NULL;
 	PHP_MD5_CTX   context;
 	unsigned char digest[16];
 
 	gettimeofday((struct timeval *) &tv, (struct timezone *) NULL);
-	sec = (int) tv.tv_sec;
-	usec = (int) (tv.tv_usec % 0x100000);
+	sec = tv.tv_sec;
+	usec = (tv.tv_usec % 0x100000);
 
 	spprintf(&tmp, 0, "%08x%05x", sec, usec);
```

Comment:
As these conversions really needed when they are just used in the spprintf call below?

Reply:
I reverted change